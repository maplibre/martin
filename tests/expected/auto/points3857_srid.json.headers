content-encoding: br
content-type: application/json
etag: W/"11c-4GaQ__e5mrlIrmds-0KbWA=="
transfer-encoding: chunked
vary: accept-encoding, Origin, Access-Control-Request-Method, Access-Control-Request-Headers
