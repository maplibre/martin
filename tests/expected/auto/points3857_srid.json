{
  "bounds": [
    -161.40590777554058,
    -81.50727021609012,
    172.51549126768532,
    84.2440187164111
  ],
  "description": "public.points3857.geom",
  "name": "points3857",
  "tilejson": "3.0.0",
  "tiles": [
    "http://localhost:3111/points3857/{z}/{x}/{y}"
  ],
  "vector_layers": [
    {
      "fields": {
        "gid": "int4"
      },
      "id": "points3857"
    }
  ]
}
